src/Forms/ConfirmedPasswordField.php

```diff
@@ -694,4 +747,17 @@ public function getRequireStrongPassword()
     {
         return $this->requireStrongPassword;
     }
+
+    private function updateRightTitle(): void
+    {
+        $text = _t(
+            __CLASS__ . '.RANDOM_IF_EMPTY',
+            'If this is left blank then a random password will be automatically generated. The user will need to click the forgotten password link to generate a reset password email so they can set their own password.'
+        );
+        if ($this->canBeEmpty && $this->generateRandomPasswordOnEmpty) {
+            $this->passwordField->setRightTitle($text);
+        } elseif ($this->passwordField->RightTitle() === $text) {
+            $this->passwordField->setRightTitle('');
+        }
+    }
```

Comment:
This will overwrite peoples' manually defined right titles. e.g:

```php
ConfirmedPasswordField::create('MyPassword')->setRightTitle('some title I want to be here')->setCanBeEmpty(true)
```

Reply:
Updated to prefix the message instead